engine/tests/ElggCoreGetEntitiesTest.php

```diff
@@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
 	}
 
 	/**
+	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
+	 */
+	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
```

Comment:
Open to suggestions.
On Aug 14, 2015 1:35 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In engine/tests/ElggCoreGetEntitiesTest.php
> <https://github.com/Elgg/Elgg/pull/8852#discussion_r37068627>:
>
> > @@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
> >  	}
> >
> >  	/**
> > +	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
> > +	 */
> > +	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
>
> Needs better function name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8852/files#r37068627>.
>